CONTRIBUTING.md

```diff
@@ -1,10 +1,18 @@
 # How to contribute
 
+This document will eventually outline all aspects of guidance to make your contributing experience a fruitful and enjoyable one.
+What it already contains is information about _commit message formatting_ and how that directly affects the numerous automated processes that are used for this repo.
+
+<table><tr><th>Table of Contents</th></tr><tr><td>
+<!-- toc -->
+<!-- tocstop -->
+</td></tr></table>
+
```

Comment:
I toyed with keeping this simpler (without the table) but found that it is much nicer with it.  This would be a standard copy-paste block that will then be filled automatically by `markdown-toc` during the release phase.